keyboards/gmmk/gmmk2/p65/ansi/rules.mk

```diff
@@ -18,5 +18,6 @@ RGBLIGHT_ENABLE = no         # Enable keyboard RGB underglow.
 AUDIO_ENABLE = no            # Audio output.
 RGB_MATRIX_ENABLE = yes      # Enable RGB matrix effects.
 RGB_MATRIX_DRIVER = AW20216  # Enable RGB matrix effects.
+RGB_MATRIX_CUSTOM_USER = yes
```

Comment:
Should never be enabled at the keyboard level, nor this file changed as part of a keymap addition. This should be instead added to a keymap level `rules.mk`
```suggestion
```

Reply:
My bad! I'll make the necessary changes.